streams/src/main/resources/common/message/SubscriptionInfo.json

```diff
@@ -35,18 +35,23 @@
     },
     {
       "name": "prevTasks",
-      "versions": "1+",
+      "versions": "1-6",
       "type": "[]TaskId"
     },
     {
       "name": "standbyTasks",
-      "versions": "1+",
+      "versions": "1-6",
       "type": "[]TaskId"
     },
     {
       "name": "userEndPoint",
       "versions": "2+",
       "type": "bytes"
+    },
+    {
+      "name": "taskOffsetSums",
+      "versions": "7+",
+      "type": "[]TaskOffsetSum"
```

Comment:
Do we want to try for a slightly more efficient encoding here, of `Map[topicGroupId -> Map[partition -> offsetSum]]`, or do you think this is fine for now?